examples/opencl2image.rs

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) 2021-2023 Via Technology Ltd. All Rights Reserved.
```

Comment:
Just 2023, the file didn't exist in 2021. ;)

Reply:
Oh just copy-pasted that. Fixed. :)